packages/beeq/src/components/page-title/bq-page-title.tsx

```diff
@@ -0,0 +1,170 @@
+  // ====================
+
+  private prefixElem: HTMLElement;
+  private suffixElem: HTMLElement;
+  private subTitleElem: HTMLElement;
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqPageTitleElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  @State() private hasPrefix = false;
+  @State() private hasSuffix = false;
+  @State() private hasSubTitle = false;
+
+  // Public Property API
+  // ========================
+
+  // Prop lifecycle events
+  // =======================
+
+  /** If true, the page title back button will be shown */
+  @Prop({ reflect: true }) haveBackNavigation: boolean;
+
+  // Events section
+  // Requires JSDocs for public API documentation
+  // ==============================================
+
+  /** Handler to be called when page title navigation button loses focus */
+  @Event() bqBlur: EventEmitter<HTMLBqPageTitleElement>;
+
+  /** Handler to be called when page title navigation button is clicked */
+  @Event() bqClick: EventEmitter<HTMLBqPageTitleElement>;
+
+  /** Handler to be called when page title navigation button is focused */
+  @Event() bqFocus: EventEmitter<HTMLBqPageTitleElement>;
+
+  // Component lifecycle events
+  // Ordered by their natural call order
+  // =====================================
+
+  // Listeners
+  // ==============
+
+  // Public methods API
+  // These methods are exposed on the host element.
+  // Always use two lines.
+  // Public Methods must be async.
+  // Requires JSDocs for public API documentation.
+  // ===============================================
+
+  // Local methods
+  // Internal business logic.
+  // These methods cannot be called from the host element.
+  // =======================================================
+
+  private handleClick = () => {
+    this.bqClick.emit(this.el);
+  };
+
+  private handleBlur = () => {
+    this.bqBlur.emit(this.el);
+  };
+
+  private handleFocus = () => {
+    this.bqFocus.emit(this.el);
+  };
+
+  private handleSlotChange = () => {
+    this.hasPrefix = hasSlotContent(this.prefixElem, 'prefix');
+    this.hasSuffix = hasSlotContent(this.suffixElem, 'suffix');
+    this.hasSubTitle = hasSlotContent(this.subTitleElem, 'sub-title');
+  };
+
+  // render() function
+  // Always the last one in the class.
+  // ===================================
+
+  render() {
+    return (
+      <div class="flex gap-xs px-[--bq-page-title--paddingX] py-[--bq-page-title--paddingY]" part="wrapper">
+        {/* Back navigation button */}
+        <div class={{ flex: true, '!hidden': !this.haveBackNavigation }} part="back">
+          <slot name="back">
+            <bq-button
+              appearance="link"
+              part="btn-back"
+              exportparts="button"
+              onBlur={this.handleBlur}
+              onClick={this.handleClick}
+              onFocus={this.handleFocus}
```

Comment:
Shouldn't the `bq events` be used here? We have bqBlur, bqClick, and bqFocus in the `bq-button` component:
https://github.com/Endava/BEEQ/blob/main/packages/beeq/src/components/button/bq-button.tsx#L111-L118

```suggestion
              onBqBlur={this.handleBlur}
              onBqClick={this.handleClick}
              onBqFocus={this.handleFocus}
```